README.md

```diff
@@ -26,6 +26,7 @@ CEP implementation for IoT platform
 -   [Data model](documentation/models.md)
 -   [Available actions for PEP](documentation/pep_actions.md)
 -   [Documentation for developers of perseo](documentation/development.md)
+-   [Development documentation (Third Iteration)](documentation/eslint.md)
```

Comment:
A new section has been added in the ToC of the README.md file, but the actual section is not there...

Reply:
I have realized it points to another file. Thus, NTC. Sorry the noise.